src/components/digitalMedia/components/digitalMedia/DigitalMediaListItem.tsx

```diff
@@ -32,45 +34,52 @@ const DigitalMediaListItem = (props: Props) => {
 
     if (specimenDigitalMedia.type === '2DImageObject') {
         digitalMediaContent = <img src={specimenDigitalMedia.mediaUrl}
-            alt={specimenDigitalMedia.id}
-            className="w-100"
+            alt={`Broken ${specimenDigitalMedia.format} link`}
```

Comment:
Yes, its valid. It will show the following text to the user if the image cannot be loaded from its url: 'Broken 2D image link'.
The user is still able to click on the text to show the metadata of the record. We can change the text later if needed or include a placeholder image.

Reply:
Makes sense :+1: